app/assets/v2/js/cart.js

```diff
@@ -497,6 +437,76 @@ Vue.component('grants-cart', {
   },
 
   methods: {
+    // Array of objects containing all donations and associated data
+    computeDonationInputs(destGitcoinAddress) {
+      let isPolygon = destGitcoinAddress == gitcoinAddressPolygon;
+
+      if (!this.grantsByTenant || this.tabSelected !== 'ETH') {
+        return undefined;
+      }
+
+      // Generate array of objects containing donation info from cart
+      const donations = this.grantsByTenant.map((grant, index) => {
+        const tokenDetails = this.getTokenByName(grant.grant_donation_currency, isPolygon);
+        let amount;
+
+        try {
+          amount = parseUnits(String(Number(grant.grant_donation_amount) || 0), tokenDetails?.decimals);
+        } catch {
+          amount = parseUnits(String(0), tokenDetails?.decimals);
+        }
+
+        return {
+          token: tokenDetails?.addr,
+          amount: amount.toString(),
+          dest: grant.grant_admin_address,
+          name: grant.grant_donation_currency, // token abbreviation, e.g. DAI
+          grant, // all grant data from localStorage
+          tokenApprovalTxHash: '' // tx hash of token approval required for this donation
+        };
+      });
+
+      // Append the Gitcoin donations (these already account for gitcoinFactor)
+      Object.keys(this.donationsToGitcoin).forEach((token) => {
+        const tokenDetails = this.getTokenByName(token, isPolygon);
+        const amount = parseUnits(String(this.donationsToGitcoin[token]), tokenDetails?.decimals);
+
+        const gitcoinGrantInfo = {
+          // Manually fill this in so we can access it for the POST requests.
+          // We use empty strings for fields that are not needed here
+          grant_admin_address: destGitcoinAddress,
+          grant_contract_address: '0xeb00a9c1Aa8C8f4b20C5d3dDA2bbC64Aa39AF752',
+          grant_contract_version: '1',
+          grant_donation_amount: this.donationsToGitcoin[token],
+          grant_donation_clr_match: '',
+          grant_donation_currency: token,
+          grant_donation_num_rounds: 1,
+          grant_id: '12',
+          grant_image_css: '',
+          grant_logo: '',
+          grant_slug: 'gitcoin-grants-official-matching-pool-fund',
+          grant_title: 'Gitcoin Grants Official Matching Pool Fund',
+          grant_token_address: '0x0000000000000000000000000000000000000000',
```

Comment:
is there any danger in burning tokens setting this address?

Reply:
@ksolo I do not think there is any danger. This is only the token address, and this address indicates that all tokens are accepted (judging by how this field is then used) see for example:
```
   const acceptsAllTokens = (grantData.grant_token_address === '0x0000000000000000000000000000000000000000' ||
      grantData.grant_token_address === '0x0');
```